packages/terra-core-docs/CHANGELOG.md

```diff
@@ -15,6 +18,7 @@
 
 * Added
   * Added an example for `terra-hyperlink` to demonstrate text ellipses when its content should overflow.
+  *
```

Comment:
nit: this extra bullet point looks like a typo?

Reply:
Good catch! removed here: 38e3192cbc5d488d06d51853a5811a1cd62dad3f